src/object/object_primitive.c

```diff
@@ -8566,7 +8566,10 @@ pr_midxkey_compare (DB_MIDXKEY * mul1, DB_MIDXKEY * mul2, int do_coercion, int t
 		    }
 		  else
 		    {
-		      c = DB_GT;
```

Comment:
Fix coding style:
1. ```c = mul1->min_max_val.type == MIN_COLUM ? DB_LT : DB_GT;```
or
2. Add brackets {} to if/else scopes.

Also, shouldn't something similar be added to ```else if (mul2->min_max_val.position == i)```?

Reply:
I'd like to propose to use `if-else` to clearly reveal the covered and uncovered branch.
Conditional statement is simple but uncertain from the point of line coverage.
I think we have to make all the branches covered. 
please also change line 8525 and 8537.